<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresenterWindow</class>
 <widget class="QWidget" name="PresenterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PresenterWindow</string>
  </property>
  <widget class="QListWidget" name="questionList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>321</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add question</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>221</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="idLine"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Text</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textLine"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Correct answer&quot;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="answerLine"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Score</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="scoreLine"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
